frontend/src/libs/components/meditation-player/components/audio-player/audio-player.tsx

```diff
@@ -0,0 +1,97 @@
+import { useCallback, useEffect, useRef, useState } from '#libs/hooks/hooks.js';
+import { type Meditation, START_TIME, STEP } from '#libs/types/types.js';
+
+import { Controls } from './controls/controls.js';
+import { ProgressBar } from './progress-bar/progress-bar.js';
+
+type Properties = {
+  src: string;
+  trackIndex: number;
+  onSetTrackIndex: (index: number) => void;
+  onSetCurrentTrack: (track: Meditation) => void;
+  tracks: Meditation[];
+};
+
+const AudioPlayer: React.FC<Properties> = ({
+  src,
+  trackIndex,
+  tracks,
+  onSetCurrentTrack,
+  onSetTrackIndex,
+}) => {
+  const [timeProgress, setTimeProgress] = useState<number>(START_TIME);
```

Comment:
we can omit this. Typescript can infer the type from the passed variable
```suggestion
  const [timeProgress, setTimeProgress] = useState(START_TIME);
```

the same in other places where primitive values were passed